templates/kratos.yaml.j2

```
@@ -5,14 +5,23 @@ identity:
     schemas:
         - id: default
           url: 'file://{{ identity_schema_file_path }}'
+{%- if default_browser_return_url or oidc_providers %}
 selfservice:
+    {%- if default_browser_return_url %}
```

This parameter is required to start kratos, so we should always add it. I think that's why the tests are failing. Besides, `oidc_providers` has no correlation in this section
```suggestion
selfservice:
```

---

To recap our conversations on mm or dailies around that topic, we should have a default url set in kratos for `default_browser_return_url`, no matter if the `login-endpoint-info` relation or its data is in place or not.

For other parameters that are not required to start kratos and run basic commands like creating identities, we can keep it as you've done in the template.

---

done